robyn/openapi.py

```diff
@@ -416,7 +416,7 @@ def override_openapi(self, openapi_json_spec_path: Path):
             self.openapi_spec = dict(json_file_content)
             self.openapi_file_override = True
 
-    def get_openapi_docs_page(self) -> FileResponse:
+    def get_openapi_docs_page(self) -> Response:
         """
         Handler to the swagger html page to be deployed to the endpoint `/docs`
         @return: FileResponse the swagger html page
```

Comment:
The problem according to mypy  is that `return html(html_file)` is a Response not a FileResponse

This would work but seems very clumsy `return FileResponse(str(html(html_file)))` 

Maybe better:
```
    def get_openapi_docs_page(self) -> FileResponse:
        """
        Handler to the swagger html page to be deployed to the endpoint `/docs`
        @return: FileResponse the swagger html page
        """
        with resources.open_text("robyn", "swagger.html") as path:
            html_file = path.read()
        return FileResponse(html_file, headers=Headers({"Content-Type": "text/html"}))
```

Reply:
@sansyrox I've committed that last idea, it returns FileResponse(...) instead of html(...)